### DISCUSSION TITLE
Scroll become lag on Mobile, when set Background-image

### DISCUSSION THREAD

**Discussion by onlyhasbi** [tailwindlabs/tailwindcss#10087] [Help]
Hello everyone, i'm facing a problem when set bg-[url('../public/image')] with .webp extension file. and open it on mobile device the scroll become lag. is there any solution to fix it ?

**Comment by ProPhycient5** [comment]
I am not sure about using `background-image` css using tailwindCSS classname. But if you convert the image into *.svg format. You can use it in inline style by following way:

```
import WelcomeBanner from "../public/icons/welcome-banner.svg";

 <div
      className="w-full h-36 px-2.5 sm:px-8 py-2 bg-contain bg-no-repeat bg-center flex flex-row justify-between items-center"
      style={{ backgroundImage: `url(${WelcomeBanner.src})` }}
    >
    </div>
    ```
